examples/c/ex_cursor.c

```diff
@@ -84,6 +85,16 @@ cursor_reset(WT_CURSOR *cursor)
 }
 /*! [cursor reset] */
 
+/*! [cursor bound] */
+int
+cursor_bound(WT_CURSOR *cursor)
+{
+    cursor->set_key(cursor, "A");
+    error_check(cursor->bound(cursor, "bound=lower,inclusive=false"));
```

Comment:
I would change this to: "action=set,bound=lower"

Reply:
[f382c49](https://github.com/wiredtiger/wiredtiger/pull/8367/commits/f382c49e92533aafae89dae772c9766b5fbc97ce)